Support for battery info and M5LED fix for M5Stickc plus2

chr0m1ng opened the pull request:
This PR adds the feature `PWRMGMT` to use `M5.Power` to get battery information for the M5Stickc plus2.
This also add/fix M5 LED for M5Stickc plus2.

n0xa commented:
Can you see if the battery level work in the develop branch works? There was an older PR covering this.

chr0m1ng commented:
@n0xa I've changed the PR to support localization and fixed set the M5LED ON/OFF values to macros as in M5StickcPlus2 if you set HIGH it will turn on (not off), which was causing the M5LED to be ON all the time
